code/game/gamemodes/blob/blobs/heatsink.dm

```diff
@@ -0,0 +1,24 @@
+/obj/effect/blob/heatsink
+	name = "heatsink blob"
+	icon_state = "blank_blob"
+	desc = "A fleshy mass capable of asborbing laser fire."
+	health = 5
+	maxhealth = 5
+	atmosblock = 1
+
+/obj/effect/blob/heatsink/update_icon()
+	overlays.Cut()
+	color = null
+	var/image/I = new('icons/mob/blob.dmi', "blob")
+	if(overmind)
+		I.color = overmind.blob_reagent_datum.color
+	src.overlays += I
+	var/image/C = new('icons/mob/blob.dmi', "blob_heatskin_overlay")
```

Comment:
The icon in the file is named blob_heatink_overlay.
I'd suggest making the color of this overlay the `overmind.blob_reagent_datum.complementary_color` and maybe greyscaling it, to make it super visually distinct from nodes